tests/modules/js-and-css/audit-enqueued-assets/audit-enqueued-assets-test.php

```diff
@@ -93,8 +93,12 @@ public function test_perflab_aea_audit_enqueued_styles_transient_already_set() {
 
 		Audit_Assets_Transients_Set::set_style_transient_with_data( 3 );
 
-		// Avoids echoing styles.
+		// Avoid deprecation warning due to related change in WordPress 6.4.
+		if ( has_action( 'wp_print_styles', 'print_emoji_styles' ) ) {
+			remove_action( 'wp_print_styles', 'print_emoji_styles' );
+		}
```

Comment:
Don't really need the conditional check. Can just call `remove_action()` directly.